.travis.yml

```diff
@@ -11,7 +11,6 @@ before_install:
   - R -q -e 'remotes::install_local(getwd(), force = TRUE); print(tic::dsl_load()); tic::prepare_all_stages()'
   - R -q -e 'tic::before_install()'
 install: R -q -e 'tic::install()'
-after_install: R -q -e 'tic::after_install()'
```

Comment:
Is this intended?

Reply:
This stage is not supported (anymore) by Travis. Discovered this by running the official Travis lintr.

Reply:
> Maintaining multiple .yml templates for each CI platform seems like a nightmare in the long run. Is there any chance to generalize this?

We could trim it down to one file for "build", "deploy" and "meta" (matrix, libs) and cat them together when building the template. This way, we would only have to maintain three files at max per CI provider.

Reply:
Sounds good.